data-prepper-plugins/cloudwatch-logs/src/main/java/org/opensearch/dataprepper/plugins/sink/config/AwsConfig.java

```diff
@@ -0,0 +1,35 @@
+package org.opensearch.dataprepper.plugins.sink.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.Size;
+import software.amazon.awssdk.regions.Region;
+
+/**
+ * AwsConfig is based on the S3-Sink AwsAuthenticationOptions
+ * where the configuration allows the sink to fetch Aws credentials
+ * and resources.
+ */
+public class AwsConfig {
```

Comment:
Let's use this and not `AwsAuthenticationOptions`

Reply:
I made the change to AwsConfig.